src/app/modules/shared/services/logging/file-logging.service.ts

```diff
@@ -129,8 +130,8 @@ export class FileLoggingService {
           deviceInfoFormatted = `manufacturer = ${deviceInfo.manufacturer}, model = ${deviceInfo.model}, os = ${deviceInfo.operatingSystem}, osVersion = ${deviceInfo.osVersion}, webViewVersion = ${deviceInfo.webViewVersion}`;
         }
       })
-      .finally(() => {
-        const versionInfo = this.appVersionService.getAppVersion();
+      .finally(async () => {
+        const versionInfo = await firstValueFrom(this.appVersionService.appVersion$);
         this.log(
           `Version = ${versionInfo.version}, build = ${versionInfo.buildNumber}, ${deviceInfoFormatted}`,
           null,
```

Comment:
Kanskje ryddigere å ikke kombinere promise og async?

Enten skrive om metoden til å bare bruke async kode, eller skrive noe a la dette hvis ikke det er så viktig at siste delen kjører i en finally-blokk:

```suggestion
          deviceInfoFormatted = `manufacturer = ${deviceInfo.manufacturer}, model = ${deviceInfo.model}, os = ${deviceInfo.operatingSystem}, osVersion = ${deviceInfo.osVersion}, webViewVersion = ${deviceInfo.webViewVersion}`;
        }
      })
      .then(firstValueFrom(this.appVersionService.appVersion$))
      .then((versionInfo) => 
        this.log(
          `Version = ${versionInfo.version}, build = ${versionInfo.buildNumber}, ${deviceInfoFormatted}`,
          null,
```

Men det er pirk, er ikke så farlig.